### DISCUSSION TITLE
Some Checkboxes are grey in the new version

### DISCUSSION THREAD

**Discussion by Frank-Seider** [OrcaSlicer/OrcaSlicer#359] [General]
In the current version, the boxes are "only one wall on the upper surface" and "detect from thin walls" grayed and can not be activated. I have a Windows 10 machine and in the previous version was all still OK. What can be the reason for this?

![BL1](https://user-images.githubusercontent.com/42341891/221206829-13ce7345-0621-4e39-a73d-3312ad3d2ed0.JPG)
![BL2](https://user-images.githubusercontent.com/42341891/221206832-320ed94b-4ed0-4753-9375-69d98821f642.JPG)

**Comment by netweaver1970** [comment]
1st example is the same as this issue... Arachne is now the default.
[https://github.com/SoftFever/BambuStudio-SoftFever/issues/340]

2nd one is also Arachne related, when switching to Classic, both become 'available'.
